authors/apps/articles/views.py

```diff
@@ -2,23 +2,38 @@
 
 from rest_framework import generics, status
 from rest_framework.response import Response
-from rest_framework.permissions import IsAuthenticated
+from rest_framework.permissions import IsAuthenticatedOrReadOnly, IsAuthenticated, AllowAny
 from rest_framework import status
 from rest_framework.views import APIView
 from rest_framework.exceptions import NotFound
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework.filters import SearchFilter
 
 from authors.apps.article_tags.views import ArticleTagView
 from authors.apps.article_tags.models import ArticleTag
+from .article_filter import ArticleFilter
 from .renderer import ArticleJSONRenderer
 from .models import Article, Like_Dislike
 from .serializers import ArticleSerializer
 from .permissions import IsOwnerOrReadOnly
 from .pagination import ArticlePageNumberPagination
 
 
-class ListCreateArticles(generics.ListCreateAPIView):
+class ListArticles(generics.ListAPIView):
```

Comment:
@mwinel If you notice the work flow was changed because there was didn't want the authentication on that end point and she chose to implement this way as the initial way wasn't working `authenticatedOrReadonly`.